obelix/idefix/devAuth/github.go

```diff
@@ -0,0 +1,43 @@
+package devAuth
+
+import (
+	"context"
+	"fmt"
+	"golang.org/x/oauth2"
+	"log"
+	"net/http"
+)
+
+func GithubAuth() *http.Client {
+	ctx := context.Background()
+	conf := &oauth2.Config{
+		ClientID:     "3569cf3d4c2b88ddc23e",
+		ClientSecret: "6d47da5ddb50575977f2f7b32381b217ae8ff785",
```

Comment:
Please consider deleting these credentials and the application in your GCP account.